lib/policies/load-balancing.js

```diff
@@ -294,15 +297,29 @@ TokenAwarePolicy.prototype.newQueryPlan = function (keyspace, queryOptions, call
  */
 function TokenAwareIterator(keyspace, queryOptions, replicas, childPolicy) {
```

Comment:
Another thing I wanted to point out was that the java driver makes whether or not to randomize replicas a configurable parameter to the `TokenAwarePolicy` constructor.  I think the reason why is initially we introduced the parameter and kept the default behavior to not shuffle the replicas, and in the next major release we made shuffling the default.  I think in hindsight that probably didn't need to be done, and if it hadn't been done it probably would have never been a parameter.  So my thoughts are:

1. Is it a concern that we are changing the behavior in a patch release?   I think it is not a concern, I don't see why the user would care that the primary replicas was always considered first and everything was always done in the same order.  It only does harm to not shuffle, and it is an implementation detail.
2. Should this be configurable?  I think it shouldn't be, similar to my thoughts about \#1.   I don't see the use case for it *not* being random.